tools/aws_lab_setup/README.md

```diff
@@ -42,6 +42,12 @@ To set up the lab for Ansible training, follow these steps.
 
         pip install boto boto3
 
+1. Install `libselinux-python`
+
+        sudo dnf install libselinux-python
```

Comment:
We aren't ready to start supporting `dnf` yet because when we do we're going to support it throughout all the content here and not just in this one doc addition.

Reply:
Is there a reason not to talk about `dnf` throughout all content? Alternatively, we could just provided that `boto`, `boto3`, and `libselinux-python` are required, and leave it up to the reader to figure out the best ways to install them.

Reply:
I'm not explicitly opposed to `dnf` -- it's starts with having the time and enough motivation (We have a lot bigger "fish to fry" as they say)  to talk about it it throughout all of the content -- decks, guides, docs etc. and be consistent.  Since this PR doesn't do that I'd prefer it be removed for now.

Generally speaking, I am also mildly concerned about the docs getting too complicated with OR... OR... OR... all over the place. I'd rather err on the side of keeping things simple and straight-forward at the expense of promoting the latest greatest thing. Those who are in-the-know will be able to adapt on their own. Not a showstopper here, but it is in the back of my mind.

Reply:
@AshtonDavis and @rubbsdecvik: I'm trying to process PRs and I'm realizing I may have lost context on this one. Why is this in the AWS setup and install guide for facilitators? Is this specific for RHEL desktop users?